scripts/release

```diff
@@ -23,7 +23,9 @@ ACTION_PATHS = [ROOT_DIR / "actions" / x / "action.yml" for x in ["iac", "secret
 CASSETTES_DIR = ROOT_DIR / "tests" / "unit" / "cassettes"
 
 # The branch this script must be run from, except in dev mode.
-RELEASE_BRANCH = "main"
+RELEASE_BRANCHES = {"main", r"\d+\.\d+\.x"}
```

Comment:
One nit: We rely on the way we use the pattern here (`fullmatch` in our case). It might be slightly more explicit to have `"^main$"` as a pattern, and use `re.match` instead of fullmatch.

Reply:
You mean `re.search()` right? because `re.match()` would only require `"main$"` 😅 (Python re module is a bit messy if you ask me...)

Going to change it.